### PR TITLE
Add solution to exercise "Understanding Work Flow...".

### PR DESCRIPTION
**Pull request by djsperka**
I added one new solution and added more description to another solution. I left three of the exercises alone and will open an issue to discuss whether they should be removed or changed.

### CODE REVIEW

**Comment by djsperka** [comment]
The new issue I refer to is #557

**Comment by kekoziar** [comment]
Hi @djsperka.  Thank you for adding solutions.  The style format is not consistent with the rest of the lesson.  References to file names and any commands - including `HEAD` - should be surrounded by the grave accent
 ( ` ). Can you edit your PR for style?

**Comment by djsperka** [comment]
Edited for style, as requested.

**Comment by kekoziar** [comment]
Verified content.  Thank you for contributing, @djsperka

File: _episodes/05-history.md [hunk 1]
```diff
@@ -363,6 +363,26 @@ moving backward and forward in time becomes much easier.
 > 4. `$ git checkout <unique ID of last commit> data_cruncher.py`
 >
 > 5. Both 2 and 4
+>
+>
+> > ## Solution
+> >
+> > The answer is (5)-Both 2 and 4. 
+> > 
+> > The checkout command restores files from the repository, overwriting the files in your working
```
Comment (nsoranzo) [inline _episodes/05-history.md]: Maybe quote `checkout` ?

Comment (kekoziar) [reply _episodes/05-history.md]: Good catch. Will you submit a PR?

Comment (nsoranzo) [reply _episodes/05-history.md]: https://github.com/swcarpentry/git-novice/pull/594

Comment (kekoziar) [reply _episodes/05-history.md]: Thanks for the contribution and quick response!

**Pull request merged**